### PR TITLE
Datastore engine module

### PR DESCRIPTION
**Pull request by nasirali1**
Data-store module to read write data/metaData

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,4 +25,7 @@ Add `pyspark.zip` to your project libraries:
 *  Now choose multiple modules / egg and "OK".
 *  Select "Classes" from categories.
 *  Give your new library a name, "My Python not Java Library"
-*  And finally click "Add Selected"
\ No newline at end of file
+*  And finally click "Add Selected"
+
+###Dependencies
+* mysql-connector-python==2.0.4
```
Comment (twhnat) [inline README.md]: Please place this in the requirements.txt file.

File: cerebralcortex/kernel/DataStoreEngine/StoreData/StoreData.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        dfData.write.format("org.apache.spark.sql.cassandra") \
+            .mode('append') \
+            .options(table=tableName, keyspace=self.keyspaceName) \
+            .save()
+
+    def datapoint(self, df: object):
+        """
+        :param df:
+        """
+        self.storeData(df, "datapoint2")
```
Comment (twhnat) [inline cerebralcortex/kernel/DataStoreEngine/StoreData/StoreData.py]: Fix this?

File: cerebralcortex/kernel/DataStoreEngine/StoreData/StoreMetadat.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2016, MD2K Center of Excellence
```
Comment (twhnat) [inline cerebralcortex/kernel/DataStoreEngine/StoreData/StoreMetadat.py]: Typo in filename.

File: .idea/CerebralCortex.iml [hunk 4]
```diff
@@ -2,16 +2,16 @@
 <module type="JAVA_MODULE" version="4">
```
Comment (twhnat) [inline .idea/CerebralCortex.iml]: Conflict

File: .idea/libraries/PySparkLib.xml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<component name="libraryTable">
```
Comment (twhnat) [inline .idea/libraries/PySparkLib.xml]: Conflict

File: .idea/misc.xml [hunk 6]
```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (twhnat) [inline .idea/misc.xml]: Conflict

File: .idea/modules.xml [hunk 7]
```diff
@@ -3,6 +3,7 @@
   <component name="ProjectModuleManager">
```
Comment (twhnat) [inline .idea/modules.xml]: Conflict

Comment (twhnat) [reply .idea/modules.xml]: Add to .gitignore

**Review by twhnat — CHANGES_REQUESTED** [review]
Needed:
* Schemas for MySQL and Cassandra
* Running example(s)

**Comment by nasirali1** [comment]
Fixed the issues and removed config files. let me know if it works.

**Pull request closed without merging**